src/sfctl/tests/version_test.py

```diff
@@ -0,0 +1,28 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""version command tests"""
+import unittest
+from subprocess import Popen, PIPE
+
+class VersionTests(unittest.TestCase):
+    """Verify current sfctl version"""
+    def test_valid_current_version(self):
+        """Tests that --version does not return error and includes the version."""
+        sfctl_version = '5.0.0'
+
+        pipe = Popen('sfctl --version', shell=True, stdout=PIPE, stderr=PIPE)
+        # returned_string and err are returned as bytes
+        (returned_string, err) = pipe.communicate()
+
+        if err:
+            err = err.decode('utf-8')
+            self.assertEqual(b'', err, msg='ERROR: in version call: ')
```

Comment:
Don't you want to fail if something gets sent to stderr?

Reply:
Yes, that would make more sense.